### ISSUE TITLE
Command line option to change output file name

### ISSUE THREAD

**Issue by jopels** [microsoft/Microsoft-Win32-Content-Prep-Tool#7]
IntuneWinAppUtil.exe names the output file after the setup file name. As Intune seems not to like it if you rename the output file manually afterwards, it would be nice to have a command line option to specify a custom name for the output file.

**Comment by laughtonsm** [comment]
I find it hard to believe that this hasn't had more attention.  This seems like it 'should' be an easy win.  I don't really want all my output files to be named 'install.intunewin'

**Comment by webash** [comment]
Agree; weird that this issue is open more than a year later...

**Comment by Tonylinden** [comment]
You can rename the output file manually after creating the intunewin-file, but I recall it can be picky about characters like dots and dashes. I usually just name it alphanumerical to be safe, like AcrobatReader15enus.intunewin etc. It´s not pretty but no-one but the admins has to see it anyway.

**Comment by rasq** [comment]
@Tonylinden yes.. you can do it manually - but its not a case. If you want to introduce some automation this is a stopper for you - easy fix in code but still: tools should have option like -pn (packagename) for user. 

For now is like 3 years. Is sad

**Comment by tony-tone** [comment]
Please could this be looked into as it would help with our package automation?

Thanks

**Comment by carrotmage** [comment]
Would also like to see this :)

**Comment by CasperStekelenburg** [comment]
+1

**Comment by rodw-wilkins** [comment]
+1

**Comment by namadori** [comment]
I can't believe this is 5 years old. It's a feature any developer should be able to implement in 10 minutes.

**Comment by sikkepitje** [comment]
I would like this too. Standardized on the same installer name for my every package : install.cmd , and so I am getting the same output file all the time. However had no trouble renaming the output file afterwards.

**Comment by DennisBergemann** [comment]
would like it

**Comment by gabrieljtaylor** [comment]
+1
